runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -304,6 +304,15 @@ void setupSwaps()
 		swap("teleport menu", "draynor village", config::swapJewelleryBox);
 		swap("teleport menu", "al kharid", config::swapJewelleryBox);
 
+		Arrays.asList(
+				"grand exchange", "varrock", "lumbridge grave'", "draynor manor", "battlefront", "mind altar",
+				"lumbridge", "falador", "salve graveyard", "camelot", "seers' village", "fenken' castle",
+				"east ardougne", "watchtower", "yanille", "senntisten", "west ardougne", "marim", "harmony island",
+				"kharyll", "lunar isle", "kourend castle", "the forgotten cemetery", "waterbirth island", "barrows",
+				"carrallangar", "fishing guild", "catherby", "annakarl", "ape atoll dungeon", "ghorrok",
+				"troll stronghold", "weiss"
```

Comment:
This doesn't seem to exactly match the teleport options available for the portal nexus. The following are the teleport options the nexus offers:

```
annakarl
ape atoll dungeon
ardougne
barrows
battlefront
camelot
carrallangar
catherby
cemetery
draynor manor
falador
fenkenstrain's castle
fishing guild
ghorrock
grand exchange
great kourend
harmony island
kharyrll
lumbridge
lumbridge graveyard
lunar isle
marim
mind altar
salve graveyard
seers' village
senntisten
troll stronghold
varrock
watchtower
waterbirth island
weiss
west ardougne
yanille
```

Notably these are the entries you have which don't match portal nexus options:
* east ardougne
* the forgotten cemetery
* fenken' castle
* kourend castle
* lumbridge grave

Reply:
Ah, I was following the wiki's listing which I think has some of them shortened for brevity, I'll get this changed!